cmake/targets/ara/apps/common/script/align_sections.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
We need to check if this is relevant. if yes, we probably need a custom cmake target which will automatically patch our linker script accordingly.

Reply:
This is used in https://github.com/pulp-platform/ara/blob/main/apps/Makefile#L48-L53
Basically the link.ld is generated by arch.link.ld as a template

Reply:
But we aren't using the Makefile right? So if we want to be able to configure the vector unit (number of labes, VLEN), we have to use a dynamically generated linker script as will, probably using cmakes `ADD_CUSTOM_COMMAND` and making the final executable depend on this.

Reply:
I will try to do it.